org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -141,4 +141,49 @@
       }
     }
   }
+
+  table {
```

Comment:
Maybe I wasn't clear before, my point was if this table styling should be applied to *all* tables or if it's just for this specific table for this specific page. If it's for all tables, then we might need to move it in a separate partial and outside the `.page-civicrm-vacancy` namespace